regtests/bin/matrix.base

```diff
@@ -2078,7 +2078,7 @@
    echo "$rtst -s OASACM  -i input_oasacm  -w work_OASACM  -C OASIS -f -p $mpi -n $np     $ww3 ww3_tp2.14" >> matrix.body
    echo "$rtst -s OASACM2 -i input_oasacm2 -w work_OASACM2 -C OASIS -f -p $mpi -n $np     $ww3 ww3_tp2.14" >> matrix.body
    halfnp=$(($np / 2))
-   echo "$rtst -s OASACM3 -i input_oasacm3 -w work_OASACM3          -f -p $mpi -n $halfnp $ww3 ww3_tp2.14" >> matrix.body
+   echo "$rtst -s OASACM3 -i input_oasacm3 -w work_OASACM3 -C OASIS -f -p $mpi -n $halfnp $ww3 ww3_tp2.14" >> matrix.body
```

Comment:
@ukmo-juan-castillo : 
Why was -C OASIS added here? Isn't OASACM3 an uncoupled run? I am getting build errors for this test.

Reply:
You are right, -C is not necessary if we have an uncoupled run. I forgot this was the case for the OASACM3 test.